### PR TITLE
wallet.sol: rename pointer variable from i to pos

### PR DESCRIPTION
**Pull request by draganm**

### CODE REVIEW

**Comment by mischat** [comment]
I am ok with this, @i-stam ?

**Comment by i-stam** [comment]
Fine with me.

**Comment by draganm** [comment]
@i-stam good point ... that's why I don't like comments :). Force pushed changes with comments renamed

File: .circleci/config.yml [hunk 1]
```diff
@@ -85,53 +85,6 @@ jobs:
             fi
           no_output_timeout: 45m
 
-  slither:
```
Comment (nostdm) [inline .circleci/config.yml]: Why did we delete the tests?

Comment (nostdm) [reply .circleci/config.yml]: @draganm It seems that you just needed to upgrade the solidity version that slither uses instead of deleting all the tests.

**Pull request merged**